manageiq-api-client.gemspec

```diff
@@ -23,12 +23,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_development_dependency "appraisal"
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "manageiq-style"
   spec.add_development_dependency "rake",          ">= 12.3.3"
   spec.add_development_dependency "rspec",         "~> 3.0"
 
-  spec.add_dependency "activesupport", ">= 5.0", "< 6.1"
+  spec.add_dependency "activesupport", ">= 5.0", "< 7.0"
```

Comment:
Can we just drop the max 7.0 statement?

do we want to test for 7.0 since we know we want to migrate there in the not too distant future?

Reply:
Well, that's the nice thing with appraisal.  We can easily add new versions by dropping these needlessly pessimistic constraints.  ActiveSupport is so stable in comparison to ActiveRecord that I would be comfortable leaving off the higher version constraint.  We can always add new appraisals for new versions if we want.

Reply:
I'm open to dropping the upper bound requirement or we can do it later.  It's not a big deal.